cmd/jaeger/internal/integration/badgercleaner/README.md

```diff
@@ -0,0 +1,47 @@
+# badger_cleaner
```

Comment:
We already discovered a similar need to have purger for ES tests. So I would like to refactor this extension so that it never mentions Badger, it should be completely generic and work with any factory. We need to add a new interface `storage.Purger` to the main storage package and do a check in this extension if the factory implements it (error out if it does not).

Does not need to be in this PR, let's first make sure Badger tests work.